### PR TITLE
Issue 889

### PR DESCRIPTION
**Pull request by dsbibby**
## Summary

Draft PR for visibility and comment while working on #889 

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [ ] Please ensure that you are making a pull request against the **Development** branch
- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [ ] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [ ] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commits related to your change
- [ ] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [ ] Please include any relevant screenshots and/or output as part of your testing
- [ ] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation changes
- [ ] Other (please describe):

## Resolved Issues

List of Issues closed or resolved by this PR. Add multiple `Closes` keyword followed by the issue number (e.g. Closes #ISSUE-NUMBER)

Closes #ISSUE-NUMBER

## Testing Verification

* Short summary of testing (e.g. successfully built and deployed VMware Event Broker Appliance)
* Please include any relevant screenshots and/or output as part of the testing

## Additional Information

* Any other details you wish to include or mention

If you have any questions/comments, feel free to reach out to team on Slack [#vcenter-event-broker-appliance](https://vmwarecode.slack.com/archives/CQLT9B5AA)

Thank you from the VEBA Team! 🥳

### CODE REVIEW

File: manual/photon.xml.template [hunk 1]
```diff
@@ -10,7 +10,7 @@
             <Label>Hostname</Label>
             <Description>Hostname (FQDN) of system</Description>
         </Property>
-        <Property ovf:key="guestinfo.ipaddress" ovf:type="string" ovf:userConfigurable="true">
+        <Property ovf:key="guestinfo.ip_address" ovf:type="string" ovf:userConfigurable="true">
```
Comment (lamw) [inline manual/photon.xml.template:13]: Just glancing over some of the changes in this PR (not a thorough review) but I was curious on why the OVF keys are being changed? The reason I ask is that there are folks who've come to rely on these for automation/scripting purposes

Comment (dsbibby) [reply manual/photon.xml.template]: After have a look I don't see any reason it shouldn't work the other way around.
I've just pushed a new commit that reverts the OVF properties back to the current development version and changes the variables throughout the setup scripts and ytt templates to match the OVF properties instead.
Is this a better approach do you think?

Comment (lamw) [reply manual/photon.xml.template]: Yup and I totally get where you were going with the consistency, especially for debugging and for new users. Appreciate the change. I'll find some time to do more thorough review and may have more questions

Comment (dsbibby) [reply manual/photon.xml.template]: 👍 Thank you.

Don't be too thorough with a review just yet mind... there's still a lot to do, but any comments on the overall approach would be great.

**Comment by dsbibby** [comment]
Update on this PR: I'll leave this draft here for now to allow further review of the changes being proposed, however I plan to submit two separate PRs to replace this one:
- one with the unit testing additions based on the current development branch to validate the unit tests against the current code base
- a second one with the script changes based on the unit test PR branch so that we can validate the script changes against the unit tests.

**Pull request closed without merging**